CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 - `DualConnection.Builder.stateListener()`
 - `StateListener`
 - `State`
+- `DualConnection.Builder.circuitBreaker()`
```

Comment:
Hmm, I thought we keep in `changelog` focused on customer,  info about api/spi,  not internals ?
Please help me calibrate my thinking.

Reply:
It was in API before the change.

Reply:
Consumers were able to call this API before. They will get compilation errors after the bump.